src/main/java/com/epam/aidial/cfg/dao/model/GlobalInterceptorEntity.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.aidial.cfg.dao.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.hibernate.envers.Audited;
+import org.jetbrains.annotations.NotNull;
+
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
+@Entity
+@Audited
+public class GlobalInterceptorEntity extends TimeTrackableEntity<String> {
+    @Id
+    @EqualsAndHashCode.Include
+    private String name;
+
+    private Integer interceptorOrder;
```

Comment:
`primary key (name, interceptor_order)` in db
Entity uses @Id on `name` only, but the database migration creates a composite primary key (name, interceptor_order). This mismatch causes JPA mapping errors

Reply:
no longer relevant